### DISCUSSION TITLE
Is there any way to use web3Auth integration with Reactjs without typescript

### DISCUSSION THREAD

**Discussion by ghost** [Web3Auth/Web3Auth#1035] [Help]
I'm using React with Javascript. Whenever I import @web3auth/modal, it gives me the error "webpack < 5 used to include polyfills". I resolved this by adding config-overrides.js file. Please guide me if I can use web3auth without typescript.

React version 18.2.0, react-scripts 5.0.1, @web3auth/modal 4.1.0-alpha.0 and web3: 1.8.1.

**Comment by atchrispinto** [comment]
Rollback web3 to 1.7.x and try

**Reply by ghost** [reply]
I'm unable to see anything in the modal. Everything is empty.

<img width="501" alt="Screenshot 2022-12-08 at 6 30 55 PM" src="https://user-images.githubusercontent.com/102660377/206459021-d9851482-8b2f-4aad-ba4a-6da3afe045e1.png">

**Reply by atchrispinto** [reply]
Rollback to 1.7.x is for the Polyfill issue

Nothing showing up - This is a web3auth issue, use a stable build rather than alpha

**Reply by ghost** [reply]
Thank you so much @chriszackpinto 🙂. Issue is resolved. There were basically versions issue with web3auth.

**Comment by tahirahmadin** [comment]
You can follow this tutorial, implementation of web3Auth in ReactJs.
Link: https://www.youtube.com/watch?v=7MkkDVLU_oY&t=127s&ab_channel=TahirAhmad

**Reply by ghost** [reply]
Hi @tahirahmadin 
I have already implemented it in ReactJS. I was getting version compatibility issues and using Alpha version so changed to the stable version. And It worked.
Thank you
